baggage/utils.py

```diff
@@ -0,0 +1,51 @@
+import math
+import itertools
+from baggage.models import Room, Bag, BAG_ADDED
+
+
+def calculate_distance(name, ini_x, ini_y, end_x, end_y):
+    return ((name, end_x, end_y), math.sqrt(pow(abs(end_x - ini_x), 2) + pow(abs(end_y - ini_y), 2)))
+
+
+def nearest_available(x, y):
+    return ''
+
+
+def get_positions(size_x, size_y, door_x, door_y):
+    return list(itertools.product(list(map(str, range(0, size_x))), list(map(str, range(0, size_y)))))
+
+
+def get_positions_dist(size_x, size_y, door_x, door_y, name):
+    positions = get_positions(size_x, size_y, door_x, door_y)
+    return list(map(lambda x: calculate_distance(name, door_x, door_y, int(x[0]), int(x[1])), positions))
+
+
+def get_all_positions(rooms):
+    return dict(map(lambda room: (room.room, get_positions(room.row, room.col, room.door_row, room.door_col)), rooms))
+
+
+def get_position(special):
+    extra = 'SPECIAL'
+    if not special:
+        rooms = Room.objects.all()
+        positions = []
+        for room in rooms:
+            positions.extend(get_positions_dist(room.row, room.col, room.door_row, room.door_col, room.room))
+        positions_sort = sorted(positions, key=(lambda x: x[1]))
+        i = 0
```

Comment:
This loop is complicated, consider using a for in or document the reason of using a while this way

Reply:
At first it had to be a `while`, but not now, changing to a for...